doc/classes/Navigation2D.xml

```diff
@@ -1,9 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Navigation2D" inherits="Node2D" category="Core" version="3.1">
 	<brief_description>
-		Class to assist with character navigation and pathfinding.
+		Node class that assists with 2D navigation and pathfinding.
```

Comment:
Saying "Node class" is redundant, plus the most important information belongs at the front.  The brief description appears next to the name in context so you just need a few words to indicate the purpose. "2D navigation and pathfinding node."